html/Mogo/styles/sections/services.scss

```diff
@@ -53,26 +58,50 @@
     height: 40px;
     transition: background 0.3s linear;
 
-
     @include respond(lg) {
       font-size: 0.9rem;
     }
   }
 
-  &__img {
-    margin: 30px 0;
-    max-width: 100%;
+  // &__img {
+  //   margin: 30px 0;
+  //   max-width: 100%;
 
-    @include respond(md) {
-      margin: 0;
-    }
-  }
+  //   @include respond(md) {
+  //     margin: 0;
+  //   }
+  // }
 
   &__panel {
     max-height: 0;
     overflow: scroll;
     text-align: left;
-    transition: max-height 0.4s linear,
-    padding 0.4s linear;
+
+    &--hidden {
```

Comment:
maybe make panel hidden by default (w/o --hidden class) and --active would show it? Because now you have combination of statuses: 

1. not hidden, not active
2. hidden, not active
3. not hidden, active
4. hidden, active

But you need only 2: hidden, not hidden OR active, not active